account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -69,15 +72,35 @@ def import_file(self):
 
     @api.model
     def _import_file(self, data_file):
-        """ Create bank statement(s) from file
-        """
+        """ Create bank statement(s) from file."""
         # The appropriate implementation module returns the required data
-        currency_code, account_number, stmts_vals = self._parse_file(data_file)
-        # Check raw data
-        self._check_parsed_data(stmts_vals)
+        statement_ids = []
+        notifications = []
+        statements = self._parse_file(data_file)
```

Comment:
@NL66278  This function's return value is different in upstream: https://github.com/OCA/bank-statement-import/blob/8.0/account_bank_statement_import/account_bank_statement_import.py#L101, https://github.com/odoo/odoo/blob/master/addons/account_bank_statement_import/account_bank_statement_import.py#L81

Is this an artifact of rebasing? We surely don't want to change this if not absolutely necessary